src/app/content/components/Header.tsx

```diff
@@ -0,0 +1,7 @@
+import styled from 'styled-components';
+
+export default styled.h1`
+  font-family: "Roboto Condensed",serif;
+  font-size: 19px;
+  line-height: 29px;
```

Comment:
should the h1 actually be styled by the cnx-recipes styles? i guess it depends on the page design.